examples/project_fully_featured/setup.py

```diff
@@ -23,10 +23,8 @@
         "dagster-pyspark",
         "dagster-slack",
         "dagster-postgres",
-        "dbt-core",
-        "dbt-duckdb",
+        "dbt-duckdb==1.3.1",  # dbt-duckdb is not yet compatible with dbt 1.4.*
```

Comment:
Why are we pinning duckdb, not dbt?

Reply:
once the dbt-duckdb adapter is updated we can remove the pin on dbt-duckdb, but if dbt-core comes first in this list then I think dbt-core will be pulled in as 1.4.x

so it's easier to just specify the pin on the package that's causing the problem